src/components/AnnualReport/AGMDate.vue

```diff
@@ -114,13 +114,15 @@
 
 <script lang="ts">
 import Vue from 'vue'
-import { Component, Mixins, Prop, Watch, Emit } from 'vue-property-decorator'
+import { Component, Prop, Watch, Emit } from 'vue-property-decorator'
 import { State, Getter } from 'vuex-class'
 import { DateMixin } from '@/mixins'
 import { FormIF } from '@/interfaces'
 
-@Component({})
-export default class AgmDate extends Mixins(DateMixin) {
+@Component({
+  mixins: [DateMixin]
+})
+export default class AgmDate extends Vue {
```

Comment:
This is the Vue 2.7+ way of using a mixin, but it only works for regular components (not mixins themselves).

(I think the `Mixins` decorator basically did the same thing on compile, but the VS Code "Volar" extension didn't recognize that, whereas it's happy with this.)